Alphabet-to-LB_readme.md

```diff
@@ -20,12 +20,14 @@ This Python script removes punctuation, converts upper to lower-case letters, ha
 
 Currently, this script is only made available as a code sample.  Many points of linguistic complexity are elided, and there is still some room to improve the accuracy of the script when it comes to acronyms and foreign words.  Please don't publish with this script!  You have been warned.

```

Comment:
I recommend moving the examples into the `Usage` section (and renaming it `Usage`), and putting `Usage` immediately after `Overview`, with `Disclaimer` following it.

In other words:
# Overview
# Usage
## Example from German
## Example from Turkish
# Disclaimer
# Attribution